### PR TITLE
Update building instructions

### PR DESCRIPTION
**Pull request by cdrage**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,48 +41,48 @@ sudo mv kompose /usr/local/bin
...
-$ go build -o kompose main.go
+```sh
+make binary
 ```
 
-If you have `go` v1.5, it's still good to build `kompose` with the following settings:
+Or `go build`:
 
-```console
-$ CGO_ENABLED=0 GO15VENDOREXPERIMENT=1 go build -o kompose main.go
```
Comment (sebgoa) [inline README.md]: we still need to keep the go1.5 instructions (this was user feedback)

Comment (cdrage) [inline README.md]: Ah, okay. I'll keep it in.

**Comment by cdrage** [comment]
@kadel @sebgoa rebased.

**Comment by cdrage** [comment]
@kadel @sebgoa @ngtuna is it possible for me to get access to add / remove labels?

File: README.md [hunk 2]
```diff
@@ -54,40 +54,46 @@ sudo mv kompose /usr/local/bin
...
 ```
 
 ## Building
 
 ### Building with `go`
 
-- You need `make`
-- You need `go` v1.6 or later.
-- If your working copy is not in your `GOPATH`, you need to set it accordingly.
+- `make`
```
Comment (sebgoa) [inline README.md]: I think you need to keep the sentences above, otherwise we have no clue what you mean.

**Comment by sebgoa** [comment]
@cdrage I will check about labels...

File: README.md (hunk 2 above)
Comment (cdrage) [inline README.md]: it looks like a requirements list, people would know what `make` is since they also see that `go` and `$GOPATH` is required.

**Comment by cdrage** [comment]
@sebgoa another review on this? :)

File: README.md (hunk 2 above)
Comment (sebgoa) [inline README.md]: looks like my review disappeared. Pretty much I disagree with you, we need empathy towards users and a few extra words goes a long way to actually tell them what to do. Making assumptions is not always good.

**Comment by sebgoa** [comment]
and you will need to rebase after editing that requirements list to what it was.

**Comment by cdrage** [comment]
Going to close this for now ^^

**Pull request closed without merging**